src/Jackalope/Transport/DoctrineDBAL/CachedClient.php

```diff
@@ -339,27 +373,11 @@ public function moveNodeImmediately($srcAbsPath, $dstAbsPath)
     /**
      * {@inheritDoc}
      */
-    public function reorderNodes($absPath, $reorders)
```

Comment:
fixed .. I added a ``clearNodeCache()`` method .. there are a few places where we could also use it if we would have access to a ``Node`` instance which I have noted in comments.